src/Message.tsx

```diff
@@ -0,0 +1,40 @@
+import { FC } from "react";
+import classnames from "classnames";
+
+import classes from "./Message.module.css";
+import ChatBubble from "./common/ChatBubble";
+import Header from "./common/Header";
+import { match, MatchConfig } from "./matcher";
+
+type MessageSource = "agent" | "user" | "bot";
+type MessageContent = any;
+
+export interface MessageProps {
+	action?: () => void;
+	className?: string;
+	message: MessageContent;
+	plugins?: MatchConfig[];
+	source: MessageSource;
+}
+
+const Message: FC<MessageProps> = props => {
+	const rootClassName = classnames(
+		"webchat-message-row",
+		props.className,
```

Comment:
className is clear, but what is the use of source here?

Reply:
Message source is used as a static class name for customizing.